og_ui/og_ui.module

```diff
@@ -1116,6 +1116,15 @@ function og_ui_get_group_admin($entity_type, $etid) {
   if (!ctype_digit($etid)) {
     return array();
   }
+  // ensure we are invoking this on something worth doing it on
+  $entity = entity_load_single($entity_type, $etid);
+  $entity_info = entity_get_info($entity_type);
+  // if this isn't a group type, skip invoking admin modules
+  if (!og_is_group_type($entity_type, $entity->{$entity_info['entity keys']['bundle']})) {
```

Comment:
thanks. Not all entities have ``$entity_info['entity keys']['bundle']``, so worth changing if to:

``if (empty($entity_info['entity keys']['bundle']) || !og_is_group_type($entity_type, $entity->{$entity_info['entity keys']['bundle']})) {``